stl/inc/yvals_core.h

```diff
@@ -1340,6 +1341,7 @@
 
 // C++23
 #if _HAS_CXX23
+#define __cpp_lib_to_underlying  202102L

```

Comment:
I'm aware I messed up the order here. Just going to wait for CI to finish before pushing.